app/jobs/wallets/refresh_ongoing_balance_job.rb

```diff
@@ -6,6 +6,8 @@ class RefreshOngoingBalanceJob < ApplicationJob
 
     unique :until_executed, on_conflict: :log
 
+    retry_on ActiveRecord::StaleObjectError, wait: :polynomially_longer, attempts: 3
```

Comment:
I can’t put my finger on it, but 3 feels low. I have no good rational reason for it though. How do you feel about 7? 

(Thinking, there is no downside to a large number here, it might avoid unexecuted jobs in case of concurrent updates)

Reply:
Agreed - We never used `7` for retry attemps in existing jobs, using `6` then 😅